Dockerfile

```diff
@@ -1,11 +1,27 @@
 FROM ubuntu:14.04
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update -q && apt-get install -y netbase python
+
+# Install ubuntu packages
+RUN apt-get update -q && \
+        apt-get install -y netbase python nginx && \
+        rm -rf /var/lib/apt/lists/*
 ADD https://raw.github.com/pypa/pip/master/contrib/get-pip.py /get-pip.py    
+
+# Install devpi and devpi-ldap via pip
 RUN python /get-pip.py
-RUN pip install "devpi-server>=2.0.6,<2.1dev" "devpi-client>=2.0.2,<2.1dev" \
-        "requests>=2.4.0,<2.5"
-VOLUME /mnt
+RUN pip install "devpi-server" \
+        "devpi-client" \
+        "requests" \
+        "devpi-ldap" \
+        "devpi-web"
```

Comment:
Pinning devpi versions is important to have reproducible builds, so please restore the original devpi package versions and we can look at upgrading them in another PR.